thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/DetectionUtils.java

```diff
@@ -219,4 +221,20 @@ public static Period getMonitoringGranularityPeriod(String monitoringGranularity
     return TimeGranularity.fromString(monitoringGranularity).toPeriod();
   }
 
+  public static Period periodFromTimeUnit(int size, TimeUnit unit) {
+    switch (unit) {
+      case DAYS:
+        return Period.days(size);
+      case HOURS:
+        return Period.hours(size);
+      case MILLISECONDS:
```

Comment:
Can you put MILLISECONDS to the end. Then the case is ordered by size.

Reply:
Or remove MILLISECONDS since it is default value.

Reply:
Sure.